x/programs/rust/examples/nft/src/lib.rs

```diff
@@ -0,0 +1,165 @@
+//! A basic ERC-721 compatible contract.
+//! The program serves as a non-fungible token with the ability to mint and burn.
+//! Only supports whole units with no decimal places.
+//!
+//! The NFT must support the common NFT metadata format.
+//! This includes the name, symbol, and URI of the NFT.
+use std::collections::HashMap;
+
+use metadata::Nft;
+use wasmlanche_sdk::{program::Program, public, state_keys, types::Address};
+
+pub mod example;
+pub mod metadata;
+
+const NAME: &str = "My NFT";
+const SYMBOL: &str = "MNFT";
+const TOTAL_SUPPLY: u64 = 20;
+
+/// The program storage keys.
+#[state_keys]
+enum StateKey {
+    /// The total supply of the token. Key prefix 0x0.
+    TotalSupply,
+    /// The name of the token. Key prefix 0x1.
+    Name,
+    /// The symbol of the token. Key prefix 0x2.
+    Symbol,
+    /// Metadata of the token. Key prefix 0x3.
+    Metadata,
+    /// Balance of the NFT token by address. Key prefix 0x4(address).
+    Balance(Address),
+    /// Counter -- used to keep track of total NFTs minted. Key prefix 0x5.
+    Counter,
```

Comment:
```suggestion
    /// Count -- used to keep track of total NFTs minted. Key prefix 0x5.
    Count,
```

Reply:
`Counter` is used in the ERC-721 contract 

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/b438cb695a1ac520cee6678610b161b1d5df4d9c/contracts/token/ERC721/presets/ERC721PresetMinterPauserAutoId.sol#L43